linera-chain/src/chain.rs

```diff
@@ -90,6 +91,8 @@ pub struct ChainStateView<C> {
     pub execution_state: ExecutionStateView<C>,
     /// Hash of the execution state.
     pub execution_state_hash: RegisterView<C, Option<CryptoHash>>,
+    /// The key sizes
+    pub key_sizes_serialized: RegisterView<C, Option<Vec<u8>>>,
```

Comment:
> the RegisterView<C, BTreeMap<Vec<u8>, usize>> does not compile with GraphQLView.

This didn't stop us in `linera-execution/src/system.rs` but ok the problem should be fixed soon with @Twey's owrk on #678, so maybe just add TODO this time.

Reply:
Also, why not MapView?

Reply:
Also, why in `ChainStateView`? We care about the storage of Wasm application only.